go-controller/pkg/clustermanager/egressip_controller_test.go

```diff
@@ -1579,91 +1579,6 @@ var _ = ginkgo.Describe("OVN cluster-manager EgressIP Operations", func() {
 			err := app.Run([]string{app.Name})
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		})
-
-		ginkgo.It("egress node update should not mark the node as reachable if there was no label/readiness change", func() {
-			// When an egress node becomes reachable during a node update event and there is no changes to node labels/readiness
-			// unassigned egress IP should be eventually added by the periodic reachability check.
-			// Test steps:
-			//  - disable periodic check from running in background, so it can be called directly from the test
-			//  - assign egress IP to an available node
-			//  - make the node unreachable and verify that the egress IP was unassigned
-			//  - make the node reachable and update a node
-			//  - verify that the egress IP was assigned by calling the periodic reachability check
-			app.Action = func(ctx *cli.Context) error {
-				config.OVNKubernetesFeature.EnableInterconnect = true // no impact on global eIPC functions
-				egressIP := "192.168.126.101"
-				nodeIPv4 := "192.168.126.51/24"
-				node := v1.Node{
-					ObjectMeta: metav1.ObjectMeta{
-						Name: node1Name,
-						Annotations: map[string]string{
-							"k8s.ovn.org/node-primary-ifaddr": fmt.Sprintf("{\"ipv4\": \"%s\"}", nodeIPv4),
-							"k8s.ovn.org/node-subnets":        fmt.Sprintf("{\"default\":[\"%s\"]}", v4NodeSubnet),
-							util.OVNNodeHostCIDRs:             fmt.Sprintf("[\"%s\"]", nodeIPv4),
-						},
-						Labels: map[string]string{
-							"k8s.ovn.org/egress-assignable": "",
-						},
-					},
-					Status: v1.NodeStatus{
-						Conditions: []v1.NodeCondition{
-							{
-								Type:   v1.NodeReady,
-								Status: v1.ConditionTrue,
-							},
-						},
-					},
-				}
-				eIP1 := egressipv1.EgressIP{
-					ObjectMeta: newEgressIPMeta(egressIPName),
-					Spec: egressipv1.EgressIPSpec{
-						EgressIPs: []string{egressIP},
-					},
-				}
-				fakeClusterManagerOVN.start(
-					&egressipv1.EgressIPList{
-						Items: []egressipv1.EgressIP{eIP1},
-					},
-					&v1.NodeList{
-						Items: []v1.Node{node},
-					},
-				)
-
-				// Virtually disable background reachability check by using a huge interval
-				fakeClusterManagerOVN.eIPC.reachabilityCheckInterval = time.Hour
-
-				_, err := fakeClusterManagerOVN.eIPC.WatchEgressNodes()
-				gomega.Expect(err).NotTo(gomega.HaveOccurred())
-
-				gomega.Eventually(getEgressIPStatusLen(eIP1.Name)).Should(gomega.Equal(1))
-				egressIPs, _ := getEgressIPStatus(eIP1.Name)
-				gomega.Expect(egressIPs[0]).To(gomega.Equal(egressIP))
-				hcClient := fakeClusterManagerOVN.eIPC.allocator.cache[node.Name].healthClient.(*fakeEgressIPHealthClient)
-				hcClient.FakeProbeFailure = true
-				// explicitly call check reachability, periodic checker is not active
```

Comment:
I assume even if we delete this test case, there is another test case that checks that the reachability functionality works correctly, right?

Reply:
yeah good point, ; let me check this

Reply:
so from checking other UTs, it seems we don't have any BUT there are e2e tests for healthchecks so we should be covered there.